docs/technologies.md

```diff
@@ -0,0 +1,423 @@
+# Technologies
+
+## Overview
+
+BigFlow provides support for the main big data data processing technologies on GCP:
+
+* [Dataflow](https://cloud.google.com/dataflow) (Apache Beam)
```

Comment:
This file is long and maybe in this place replace the external links for links linking to the pargagraphs in this file and in paragraph add link to specific technology

Reply:
done